web/src/components/studies/Studies.tsx

```diff
@@ -1,18 +1,35 @@
 import { useAuth } from "@/hooks/useAuth";
 import LoginFallback from "@/components/ui/LoginFallback";
-import Title from "@/components/ui/Title";
+import ApprovedResearcherView from "./ApprovedResearcherView";
+import Button from "../ui/Button";
 
 export default function Studies() {
-  const { authInProgress, isAuthed } = useAuth();
-
+  const { authInProgress, isAuthed, userData } = useAuth();
   if (authInProgress) return null;
 
   if (!isAuthed) return <LoginFallback />;
 
+  const isApprovedResearcher = userData?.roles.includes("approved-researcher");
+  const isAdmin = userData?.roles.includes("admin");
+
+  const username = userData!.username;
+
   return (
     <>
-      <Title text={"Studies"} />
-      <p>This page is being built. Please check back soon for updates!</p>
+      {!isAdmin && !isApprovedResearcher && (
+        <>
```

Comment:
This logic means that if I am an admin but not an approved researcher I will not see the text block. Maybe just remove the `!isAdmin` check here, unless it was added for a reason?

Reply:
fair point, I'll remove it for now; there will be an admin view later (view all studies) but I suppose I jumped the gun